.vscode/extensions.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Should we be committing the `.vscode` folder? Won't this cause each of our local VS Code options to conflict every time we add and push changes?

Reply:
I really don't know. Is this file shared by all extensions? I put it here because in the template it was present.

Reply:
I think this is something that we each need to set up locally, and we need a `.gitignore` file to ignore the `.vscode` directory. It stores local (personal) preferences/config, so I don't think it shouldn't be committed.